<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StockViewerDialog</class>
 <widget class="QDialog" name="StockViewerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>875</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="labelTicker">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Ticker</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>118</width>
         <height>18</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <property name="currentIndex">
        <number>12</number>
       </property>
       <item>
        <property name="text">
         <string>30 minutes</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1 hour</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3 hours</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6 hours</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>12 hours</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1 day</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3 days</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1 week</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2 weeks</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1 month</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3 months</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6 months</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1 year</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3 years</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5 years</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>max</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTableWidget" name="tableWidget">
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustToContents</enum>
       </property>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="PlotWidget" name="graphicsView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>StockViewerDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>StockViewerDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
